recipes/embree3/all/conanfile.py

```diff
@@ -170,6 +170,13 @@ def _configure_cmake(self):
         return self._cmake
 
     def build(self):
+        # some compilers (e.g. clang) do not like UTF-16 sources
```

Comment:
for an additional safety, we may put it under `if self.settings.compiler == "clang'`. but I've checked, and Visual Studio builds just fine with it.